CARET_trace/CMakeLists.txt

```diff
@@ -51,6 +51,22 @@ ament_target_dependencies(caret
   rmw_fastrtps_shared_cpp
   LTTngUST)
 
+add_library(caret_test_core SHARED
```

Comment:
Could you tell me  why `caret_test_core` is needed ?
Isn't it possible to test against `caret` ?

Reply:
Though I was thinking to disable influence of hook codes  (e.g. ros_trace_points.cpp and hooked_trace_points.cpp) , creating caret_test_core wasn't make sense.

I removed caret_test_core in: [28df784](https://github.com/tier4/CARET_trace/pull/46/commits/28df784614ce61304ecde3659189ab595b7c06ea)

Reply:
Understood. Thank you for the explanation.